build-system/tasks/storybook-react/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "amp-storybook",
```

Comment:
This is sharing a name with build-system/tasks/storybook/package.json. Maybe change it to include "react"?

(Unless you agree with my first comment, in which case this file can be removed.)